src/lib/event/content/Kind30023.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+    import type { NDKArticle } from "@nostr-dev-kit/ndk";
+    import type NDK from "@nostr-dev-kit/ndk";
+    import NoteContentNewline from "./NoteContentNewline.svelte";
+    import NoteContentTopic from "./NoteContentTopic.svelte";
+    import NoteContentLink from "./NoteContentLink.svelte";
+
+    import EventCard from "../EventCard.svelte";
+    import NoteContentPerson from "./NoteContentPerson.svelte";
+    import { LINK, HTML, NEWLINE, TOPIC, parseContent } from "../../utils/notes.js";
+    import { markdownToHtml } from "$lib/utils/markdown";
+
+    export let ndk: NDK;
+    export let article: NDKArticle;
+    export let showMedia: boolean = true;
+    export let anchorId: string | null = null;
+
+    const htmlContent = markdownToHtml(article.content);
+    const parsed = parseContent({ content: htmlContent, tags: article.tags, html: true });
+</script>
+
+<div class="article">
+    <h1 class="article-title">{article.title}</h1>
```

Comment:
I think this doesn't belong here; we should just let the user render the content on it's own. The title is too trivial to render outside the component but not possible to not-render.

Reply:
fair – update incoming.